packages/ui/src/components/filter/components/filter-buttons/filter-buttons.styles.ts

```diff
@@ -1,5 +1,19 @@
 import { tv } from 'tailwind-variants';
 
 export const styles = tv({
-  base: 'flex w-full gap-1 overflow-auto whitespace-nowrap',
+  slots: {
+    container: 'relative flex justify-start',
+    scrollButtonLeft:
```

Comment:
When the button is pressed the active class is still being applied so a dark background appears not sure if this is the intended design. Might need to check with GEL team

Reply:
this is a styling error, will restore to white